libcxx/test/libcxx/clang_modules_include.gen.py

```diff
@@ -49,6 +49,9 @@
 // TODO: Investigate why this doesn't work on Picolibc once the locale base API is refactored
 // UNSUPPORTED: LIBCXX-PICOLIBC-FIXME
 
+// TODO: Fix seemingly circular inclusion or <wchar.h> on AIX
+// UNSUPPORTED: LIBCXX-AIX-FIXME
```

Comment:
@daltenty This appears to start failing on AIX after this patch, see e.g. https://buildkite.com/llvm-project/libcxx-ci/builds/39304#0193b131-ead6-42af-b231-48d8de48b2a6. I don't understand why it fails though, since we have proper include guards in `wchar.h` around the functions that are seemingly re-defined. Since I don't have access to an AIX box matching the CI to reproduce, it would be helpful if someone on the AIX side could take a look.